client/client.go

```diff
@@ -0,0 +1,768 @@
+package client
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/xml"
+	"fmt"
+	"io"
+	urlPkg "net/url"
+	"os"
+	"path/filepath"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/gofiber/fiber/v3/log"
+	"github.com/gofiber/utils/v2"
```

Comment:
V2 or v3?

Reply:
Utils package's versioning is not same core repo